debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4

```diff
@@ -2010,6 +2012,12 @@ engineName
     | CONNECT
     ;
 
+// MariaDB
+encryptedLiteral
+    : '`' ENCRYPTED '`' 
+    | ENCRYPTED
+    ;
```

Comment:
```suggestion
encryptedLiteral
    : ENCRYPTED
    | STRING_LITERAL
    ;
```
@ismailsimsek You should use the `STRING_LITERAL`, that's what @Naros implied. This would help in encountering the literals with backticks.